lib/ops/opsservice/versions.go

```diff
@@ -59,11 +59,8 @@ var (
 	// For instance, if the version is 5.2.10, the current version can upgrade
 	// directly from 5.2.10, 5.2.11 and so on.
 	DirectUpgradeVersions = Versions{
-		semver.New("6.1.0"),
-		semver.New("6.2.0"),
-		semver.New("6.3.0"),
-		semver.New("7.0.0"),
 		semver.New("7.1.0"),
```

Comment:
It'd probably be worth leaving a comment about why 7.1 is supported -- those tags will become an odd historical artifact.